test/screenshot/spec/mdc-snackbar/classes/wide.html

```diff
@@ -50,7 +50,6 @@
            id="test-snackbar--wide"
            role="status"
            aria-live="polite"
-           aria-hidden="true"
```

Comment:
You've been moving the role/aria-live attributes to the label but you didn't do that here?

(I'm actually curious why we want these attributes only on the label - does that mean the action/dismiss buttons won't be announced? How will a user relying on screen readers know that they exist?)

Reply:
Thanks, fixed!

You are correct, the action/dismiss buttons won't be announced.

According to [WCAG 2.0 — SC 2.2.3 — No Timing](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html), UI controls should not be timing-dependent. We don't want to announce a button that won't be there by the time the user navigates to it.

The snackbar a11y spec also says that apps should provide a redundant means of achieving the same action elsewhere in the app.

So for AT users, this is simply a passive notification.